edge/pkg/edged/volume/csi/csi_client.go

```diff
@@ -217,7 +217,7 @@ func (c *csiDriverClient) NodeGetInfo(ctx context.Context) (
 		var getNodeInfoError error
 		if c.nodeV1ClientCreator != nil {
 			nodeID, maxVolumePerNode, accessibleTopology, getNodeInfoError = c.nodeGetInfoV1(ctx)
-		} else if c.nodeV1ClientCreator != nil {
```

Comment:
`} else if c.nodeV0ClientCreator != nil {` ?

Reply:
but the struct have no the field `c.nodeV0ClientCreator`
https://github.com/kubeedge/kubeedge/blob/f7c9abcebc5cf6b3825785b9dfffec9977bec55f/edge/pkg/edged/volume/csi/csi_client.go#L109-L114

git commit 56f91bf6b3bf2e613b7ab3af2df93b692b1cf3ff remove this field `nodeV1ClientCreator`

My guess is that PR didn't delete this else if statement when we updated the code in bulk

The V0 version is no longer in use

Reply:
Maybe it needs to be cleaned up again

Reply:
Looks Good.